src/main/java/com/palantir/docker/compose/DockerComposeRule.java

```diff
@@ -99,10 +104,17 @@ public void before() throws IOException, InterruptedException {
     @Override
     public void after() {
         try {
-            log.debug("Killing docker-compose cluster");
-            dockerCompose().down();
-            dockerCompose().kill();
-            dockerCompose().rm();
+            if (skipShutdown()) {
+                log.error("******************************************************************************************\n"
+                        + "******** Skipping docker-compose down/kill/rm; This should not happen on CI! *************\n"
```

Comment:
Can you make it clear that the rule has been configured this way, it's not some weird docker error.